pliers/extractors/text.py

```diff
@@ -402,3 +401,147 @@ def _extract(self, stim):
+            model_kwargs = {}
+
+        if tokenizer_kwargs is None:
+            tokenizer_kwargs = {}
+
+        self.pretrained_model = pretrained_model_or_path
+        self.tokenizer_type = tokenizer
+        self.framework = framework
+        self.encoding_level = encoding_level
+        self.pooling = pooling
+        self.model_kwargs = model_kwargs
+        self.tokenizer_kwargs = tokenizer_kwargs
+        model_name = 'BertModel' if self.framework == 'pt' else 'TFBertModel'
+
+        self.tokenizer = transformers.BertTokenizer.from_pretrained(
+            tokenizer, **tokenizer_kwargs)
+        self.model = getattr(transformers, model_name).from_pretrained(
+            pretrained_model_or_path, **model_kwargs)
+
+        super(PretrainedBertEncodingExtractor, self).__init__()
+
+    def _extract(self, stims):
+
+        if stims.name == '':
+            stims.name = ' '.join([s.text for s in stims.elements])
+
+        text, onsets, durations = zip(
+            *((s.text, s.onset, s.duration) for s in stims.elements))
+        tokens = [self.tokenizer.tokenize(t) for t in text]
+        tokens_flat = list(flatten(tokens))
+
+        def cast_to_token_level(word_level_list):
+            token_level_list = [itertools.repeat(word_level_list[i], len(tok))
+                                for i, tok in enumerate(tokens)]
+            token_level_list = list(flatten(token_level_list))
+            return token_level_list
+
+        t_text, t_ons, t_dur = map(cast_to_token_level,
+                                   [text, onsets, durations])
+        tensor_tokens = self.tokenizer.encode(tokens_flat,
+                                              return_tensors=self.framework)
+        output = self.model(tensor_tokens)
+        output = [out.detach().numpy() if self.framework == 'pt' else
+                  out.numpy() for out in output]
+
+        if self.encoding_level == 'token':
+            encoded_tokens = tokens_flat
+            encodings = output[0][:, 1:-1, :].squeeze()
+
+        elif self.encoding_level == 'sequence':
+            encoded_tokens = [' '.join(text)]
+            t_text = ['None']
+            if not any(val is None for val in [t_ons[-1],t_dur[-1],t_ons[0]]):
+                t_dur = t_ons[-1] + t_dur[-1] - t_ons[0]
+            else:
+                t_dur = None
+            t_ons = t_ons[0]
+
+            if self.pooling:
+                pooling_function = getattr(np, self.pooling)
+                encodings = pooling_function(output[0][:, 1:-1, :],
+                                             axis=1, keepdims=True)
+            else:
+                encodings = output[1]
+
+        data = [encodings.tolist(), encoded_tokens, t_text]
+        features = ['encoding', 'token', 'word']
+
+        return ExtractorResult(data, stims, self,
+                               features=features,
+                               onsets=t_ons, durations=t_dur)
+
+    def _to_df(self, result, model_attributes=True):
```

Comment:
```suggestion
    def _to_df(self, result, model_attributes=False):
```

Reply:
Would lean towards leaving default to `True`. I would expect that in most cases you would want to know which token is being encoded.

Reply:
OK, I don't feel strongly about this, especially once my PR is merged.